README.md

```diff
@@ -10,7 +15,7 @@ Pre-requisites: You must have Python 3.7 or above installed on your computer.
     ```bash
     python3 -m venv .venv
     source .venv/bin/activate
-    pip install --editable .
+    pip install evolv
```

Comment:
Do we want to name it to something like evolv-experiments-sdk since we might release other evolv libraries?